src/eyeTracker.m

```diff
@@ -55,161 +30,134 @@
-                % Eyelink('command', 'generate_default_targets = NO');
-                %
-                % % calibration and validation target locations
-                % [width, height]=Screen('WindowSize', screenNumber);
-                % Eyelink('command','calibration_samples = 6');
-                % Eyelink('command','calibration_sequence = 0,1,2,3,4,5');
-                % Eyelink('command','calibration_targets = %d,%d %d,%d %d,%d %d,%d %d,%d',...
-                % 640,512, ... %width/2,height/2
-                % 640,102, ... %width/2,height*0.1
-                % 640,614, ... %width/2,height*0.6
-                % 128,341, ... %width*0.1,height*1/3
-                % 1152,341 ); %width-width*0.1,height*1/3
-                %
-                % Eyelink('command','validation_samples = 5');
-                % Eyelink('command','validation_sequence = 0,1,2,3,4,5');
-                % Eyelink('command','validation_targets = %d,%d %d,%d %d,%d %d,%d %d,%d',...
-                % 640,512, ... %width/2,height/2
-                % 640,102, ... %width/2,height*0.1
-                % 640,614, ... %width/2,height*0.6
-                % 128,341, ... %width*0.1,height*1/3
-                % 1152,341 ); %width-width*0.1,height*1/3
-                % % % % % % % % % % % % % % % % % % % % % % % % % % % % % %
-
-                % % set parser (conservative saccade thresholds)
-                % Eyelink('command', 'saccade_velocity_threshold = 35');
-                % Eyelink('command', 'saccade_acceleration_threshold = 9500');
-
-                % set EDF file contents (not clear what this lines are used for)
-                el.vsn = regexp(el.vs, '\d', 'match'); % wont work on EL
+                    % Set custom calibration parameters.
+                    Eyelink('command', 'generate_default_targets = NO');
 
-                % enter Eyetracker camera setup mode, calibration and validation
-                EyelinkDoTrackerSetup(el);
+                    % Calibration target locations, set manually the dots
+                    %  coordinates, here for 6 dots.
 
-                % % do a final check of calibration using driftcorrection
-                % % You have to hit esc before return.
-                % EyelinkDoDriftCorrection(el);
+                    % [width, height]=Screen('WindowSize', screenNumber);
 
-                % % do a final check of calibration using driftcorrection
-                % success=EyelinkDoDriftCorrection(el);
-                % if success~=1
-                % Eyelink('shutdown');
-                % cleanUp()
-                % return;
-                % end
+                    Eyelink('command', 'calibration_samples = 6');
+                    Eyelink('command', 'calibration_sequence = 0,1,2,3,4,5');
+                    Eyelink('command', 'calibration_targets = %d,%d %d,%d %d,%d %d,%d %d,%d', ...
+                        640, 512, ... % width/2,height/2
```

Comment:
We definitely want to get rid of that hard coding and be able to set that up from a setParameters file